tests/test_graphs.py

```diff
@@ -116,7 +116,7 @@ def test_get_heatflow_graph(mc):
 
     assert len(x) == len(y) == 9
     assert len(x1) == len(y1) == 11
-    assert almost_equal(y1[3], 11.68)
+    assert almost_equal(y1[3], 12.45)
```

Comment:
I am thinking we could make this more generic and do something like just check y1[3] > y1[4]? My concern is that the results may change due to calculation setting changes in future releases of the software, and we don't want to have to keep updating the values.

Reply:
Agreed, I've done this and commented about why this makes sense for this series.

Reply:
I have also changed the other tests so they use the equivalent graph_point functions, and cross check the results. This isn't possible in this case, as there isn't a get_heatflow_graph_point function.